### PR TITLE
Document additional issue management process

### PR DESCRIPTION
**Pull request by theunrepentantgeek**
**What this PR does / why we need it**:

Updates our documented issue mangement process to cover more than just new issue triage.

**How does this PR make you feel**:
gif

**If applicable**:
- [x] this PR contains documentation

### CODE REVIEW

**Review by babbageclunk — APPROVED** [review]

File: docs/issue-management.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+```
+is:issue is:open label:needs-triage
+```
+
+When new issues are created, our bot will automatically add the `needs-triage` tag so they show up in this list.
+
+## Old Issue Review
+
+On a regular basis (target: weekly) the development team will review old issues using the folowing [GitHub search]():
```
Comment (Porges) [inline docs/issue-management.md:15]:
```suggestion
On a regular basis (target: weekly) the development team will review old issues using the following [GitHub search]():
```

Comment (Porges) [reply docs/issue-management.md:15]: Also missing link

**Review by Porges — CHANGES_REQUESTED** [review]

File: docs/issue-management.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Issue Management
+
+## New Issue Triage
+
+On a regular basis (target: weekly) the development team will triage new issues matching the following [GitHub search](https://github.com/Azure/azure-service-operator/issues?q=is%3Aissue+is%3Aopen+label%3Aneeds-triage+):
+
+```
+is:issue is:open label:needs-triage
```
Comment (babbageclunk) [inline docs/issue-management.md:8]:
```suggestion
is:issue is:open label:"needs-triage :mag:"
```

Comment (theunrepentantgeek) [reply docs/issue-management.md]: Fixed.

File: docs/issue-management.md (hunk 1 above)
Comment (theunrepentantgeek) [reply docs/issue-management.md]: Fixed.

**Review by Porges — APPROVED** [review]

**Pull request merged**